### DISCUSSION TITLE
Welcome to discuss i Candy Robot 欢迎讨论爱糖机

### DISCUSSION THREAD

**Discussion by emicoto** [emicoto/DOLMods#1] [Announcements]
大家可以到这里反馈爱糖机的bug与体验
Please report bugs and experience at here!

**Comment by STH-SKYLIGHT** [comment]
bug求助 
在游泳馆被小出生打了一针叫天使醉的玩意之后打架打出去了，然后又折腾半天得了个成瘾。就是在这时候发现左侧的物品栏按钮不见了，在选项和作弊里找了好久也没能恢复这个按钮，去百货大楼买东西也没显示，已经饿晕好几次了，咋办啊（
